#include <exception>
#include "../utils/test_utils.hpp"

int main()
{
    test_assert(true);

    return 0;
}